platforms/quarkus/api/README.md

```diff
@@ -0,0 +1,30 @@
+# apicurio-studio-platforms-quarkus-api project
+
+This project uses Quarkus, the Supersonic Subatomic Java Framework.
+
+If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
+
+## Running the application in dev mode
+
+You can run your application in dev mode that enables live coding using:
+```
+./mvnw quarkus:dev
```

Comment:
Can you change all the references in this Readme from `./mvnw` to just `mvn`?  I think it works with standard maven, and we aren't using mvnw in this project ATM.

Reply:
Sure.